examples/clkgen_in_hdl/test_clkgen.py

```diff
@@ -0,0 +1,74 @@
+# This file is public domain, it can be freely copied without restrictions.
+# SPDX-License-Identifier: CC0-1.0
+
+import cocotb
+from cocotb.clock import Clock
+from cocotb.triggers import Timer
+
+MAXCOUNT = 123456
```

Comment:
This is a good case for a plusarg or environment variable communicating to cocotb.